src/analytics/preferences/usePreferencesAnalytics.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Analytics as A,
+  Properties,
+  addPageAction,
+} from "analytics/addPageAction";
+import { FilterLogic } from "constants/enums";
+
+type Action =
+  | { name: "Open Task Link" }
+  | { name: "Open Job Logs" }
+  | { name: "Open Raw Logs" }
+  | { name: "Open HTML Logs" }
+  | { name: "Clicked Copy To Jira" }
+  | { name: "Toggle Wrap"; on: boolean }
+  | { name: "Toggle Case Sensitivity"; insensitive: boolean }
```

Comment:
nit. I think you can also use `on` to refer to this. "Case sensitivity is on" / "Case sensitivity is off" makes sense to me
```suggestion
  | { name: "Toggle Case Sensitivity"; on: boolean }
```